File: backend/src/main/java/com/krushisevakendra/billing/auth/dto/LoginResponse.java
package com.krushisevakendra.billing.auth.dto;

import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class LoginResponse {

    private String accessToken;

    private String tokenType;

    private String username;

    private String role;
}

File: mobile/lib/models/login_response.dart

class LoginResponse {

  final String accessToken;
  final String tokenType;
  final String username;
  final String role;

  LoginResponse({
    required this.accessToken,
    required this.tokenType,
    required this.username,
    required this.role,
  });

  factory LoginResponse.fromJson(Map<String,dynamic> json){
    return LoginResponse(
      accessToken: json['accessToken'],
      tokenType: json['tokenType'],
      username: json['username'],
      role: json['role'],
      );
  }
}